Patch was imported from the OpenEmbedded git server
(git://git.openembedded.org/openembedded)
as of commit id 8f491bf275cf8a64d05e99f2e5b0c6c5143202b4
Signed-off-by: Henning Heinold <heinold@inf.fu-berlin.de>

Index: klibc-1.5.15/klcc/klcc.in
===================================================================
--- klibc-1.5.15.orig/klcc/klcc.in	2009-12-14 00:32:41.373661102 +0100
+++ klibc-1.5.15/klcc/klcc.in	2009-12-14 00:34:20.855735356 +0100
@@ -147,7 +147,7 @@
     } elsif ( $a =~ /^-([fmwWQdO]|std=|ansi|pedantic|M[GPD]|MMD)/ ) {
	# Options to gcc
	push(@ccopt, $a);
-    } elsif ( $a =~ /^-([DUI]|M[FQT])(.*)$/ ) {
+    } elsif ( $a =~ /^-([DUI]|M[FQT]|isystem)(.*)$/ ) {
	# Options to gcc, which can take either a conjoined argument
	# (-DFOO) or a disjoint argument (-D FOO)
	push(@ccopt, $a);
